Source: maxice8
Upstream: no, it seems to be inactive.
Reason: fixes usage of dmraid.h under musl libc.

--- 1.0.0.rc16-3/dmraid/include/dmraid/misc.h
+++ 1.0.0.rc16-3/dmraid/include/dmraid/misc.h
@@ -10,6 +10,7 @@
 
 #ifndef _MISC_H_
 #define _MISC_H_
+#define _GNU_SOURCE
 #include <fcntl.h>
 
 #define DM_ASSERT(__cond) do { if (!(__cond)) { printf("ASSERT file:%s line:%d fuction:%s cond: %s\n", __FILE__, __LINE__, __FUNCTION__, #__cond); } } while(0);
